concrete/src/Package/StartingPointPackage.php

```diff
@@ -416,9 +416,10 @@ protected function install_api()
     {
         $scopes = $this->app->make('config')->get('app.api.scopes');
         $em = $this->app->make(EntityManager::class);
-        foreach($scopes as $scope) {
```

Comment:
I think you could make this backwards compatible by checking if `$scope` is numeric and using $description as the scope instead